### DISCUSSION TITLE
How to prevent Xiaomi devices to force Dark Mode in my apps?

### DISCUSSION THREAD

**Discussion by PabloMzGa** [dotnet/maui#9634] [Q&A]
Hi everyone. Honestly I am still a junior developer and this is the first time I post something in a public Github, so I am not so sure how to explain or show all the necesary data.

The thing is, I am trying to create a simple Android app and I have just configured the themes using AppThemeinding in AppShell.xaml and it works good in other devices (I have checked in OnePlus t8 and Oppo A5), but in Xiaomi devices (Mi10 with MIUI 12.5.5 and in PocoX3 NFC with MIUI 12.5.8), the app colors will change, no matter if I specify them with AppThemeBinding or if i use UserAppTheme = "Light".

I just want the background colors to stay the same. Is this a bug, or anyone has idea about how to solution this?

Thanks

PS: Just updated the Mi10 to MIUI 13.0.2 and it works fine.

**Comment by Eatham532** [comment]
Try to edit `/Resources/Styles/Colors.xaml`

**Reply by PabloMzGa** [reply]
I deleted the reference to both Colors.xaml and Styles.xaml in App.xaml and it still happens

**Reply by Eatham532** [reply]
Those files specify which colors are used. You could specify the dark mode colors to be used for light mode.

**Reply by PabloMzGa** [reply]
I know, I used my own files and it works in the others devices I checked. It's a Xiaomi problem but I was wondering if there was a workaround

**Reply by jayasankarC1** [reply]
If your Xiaomi device in dark mode, it has some problems. The app forcefully change into dark, We can turn off on settings. There are many related issues being reported in Dark mode.
